test/module/shared_model/validators/validators.hpp

```diff
@@ -18,11 +18,9 @@
 #ifndef IROHA_VALIDATOR_MOCKS_HPP
 #define IROHA_VALIDATOR_MOCKS_HPP
 
-#include <gmock/gmock.h>
-
-#include "interfaces/iroha_internal/block.hpp"
 #include "validators/abstract_validator.hpp"
-#include "validators/answer.hpp"
+
+#include <gmock/gmock.h>
```

Comment:
Third party libraries should be above ours

Reply:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

>Use standard order for readability and to avoid hidden dependencies: Related header, C library, C++ library, other libraries' .h, your project's .h.

`abstract_validator.hpp` is the related header for this file.